src/main/resources/com/google/api/codegen/java/http_stub.snip

```diff
@@ -53,7 +53,8 @@
 @end
 
 @private httpMethodCallable(methodDescriptor)
-  private static final ApiMethodDescriptor<{@methodDescriptor.requestTypeName}, {@methodDescriptor.responseTypeName}> {@methodDescriptor.name} =
+  @@VisibleForTesting
```

Comment:
We shouldn't be using `@VisibleForTesting` (which is from Guava), we should be using `@InternalApi` from api-common

Reply:
ahh sure sure